spam/spam_tutorial.py

```diff
@@ -540,27 +580,47 @@ def plot_label_frequency(L):
 label_model.fit(L_train, n_epochs=500, log_freq=50, seed=123)
 
 # %%
-print(
-    f"{'Majority Vote:':<14} {majority_model.score(L_valid, Y_valid)['accuracy']:0.3f}"
-)
-print(f"{'Label Model:':<14} {label_model.score(L_valid, Y_valid)['accuracy']:0.3f}")
+majority_acc = majority_model.score(L_valid, Y_valid)["accuracy"]
+print(f"{'Majority Vote Accuracy:':<25} {majority_acc * 100:.1f}%")
 
+label_model_acc = label_model.score(L_valid, Y_valid)["accuracy"]
+print(f"{'Label Model Accuracy:':<25} {label_model_acc * 100:.1f}%")
 
 # %% [markdown]
-# While our `LabelModel` does improve over the majority vote baseline, it is still somewhat **limited as a classifier** for the reasons outlined in the Snorkel 101 guide.
-#
-# In the next section, we will use these training labels to train a discriminative classifier to see if we can improve performance further.
+# So Our `LabelModel` improves over the majority vote baseline!
+# However, it is typically **not suitable as an inference-time model** to make predictions for unseen examples.
```

Comment:
Unlike in crowdsourcing, LFs in this case can be applied to real examples. So I assume the reason is primarily lack of coverage? Might be word adding a few words to this line.